src/docs/reference/modules/Elasticsearch/README.md

```diff
@@ -55,20 +55,24 @@ Here is a sample step:
 
 ```json
 {
-  "name": "ElasticIndexSettings",
-  "Indices": [
-    {
-      "Search": {
-        "AnalyzerName": "standardanalyzer",
-        "IndexLatest": false,
-        "IndexedContentTypes": [
-          "Article",
-          "BlogPost"
-        ]
-      }
-    }
-  ]
-},
+    "steps":[
+        {
+            "name":"ElasticIndexSettings",
+            "Indices":[
+                {
+                    "Search":{
+                        "AnalyzerName":"standardanalyzer",
+                        "IndexLatest":false,
+                        "IndexedContentTypes":[
+                            "Article",
+                            "BlogPost"
+                        ]
+                    }
+                }
+            ]
+        }
+    ]
+}
```

Comment:
Note the added spaces after each colon. This is how recipes are generated so the docs should reflect that too. This also applies to all the other changes.

```suggestion
    "steps": [
        {
            "name":" ElasticIndexSettings",
            "Indices": [
                {
                    "Search": {
                        "AnalyzerName": "standardanalyzer",
                        "IndexLatest": false,
                        "IndexedContentTypes": [
                            "Article",
                            "BlogPost"
                        ]
                    }
                }
            ]
        }
    ]
}
```

Reply:
@Piedone done.